### PR TITLE
Gdr 2387

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2387 

## Why was it changed?
To create vignette and visualization for a data.model

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by marcin-kam — APPROVED** [review]

File: vignettes/gDR-data-model.Rmd [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+2. Controls -- represents untreated, control references
+3. Normalized -- represents normalized data to compute RelativeViability and GRValues (default gDR normalization types)
+4. Averaged -- stores averaged replicates computed by the mean and standard deviation
+5. Metrics -- contains fitted response curves
+6. excess (*) -- excess data for each pair of concentration values (represents Bliss excess, HSA excess, and data smoothing values)
+7. all_iso_points (*) stores isobologram points
+8. isobolograms (*) -- stores isobologram curves
+9. scores (*) -- scores data for each pair of concentration values (HSA score, Bliss Score, and CI (combination index) scores)
+
+All assays are represented by `r Biocpkg("BumpyMatrix")` objects.
```
Comment (gladkia) [inline vignettes/gDR-data-model.Rmd:83]:
```suggestion
All assays are stored as `r Biocpkg("BumpyMatrix")` objects.
```

File: vignettes/gDR-data-model.Rmd [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+To extract specific experiments from the `r Biocpkg("MultiAssayExperiment")` object, the `[[` operator can be used For example, to access the data related to combination experiments, one can use `MAE[["combination"]]`, where `MAE` represents the `r Biocpkg("MultiAssayExperiment")` object.
+
+To gain insights into the available experiments within the `r Biocpkg("MultiAssayExperiment")` object, the `MultiAssayExperiment::experiments` function can be used.
+
+
+# SummarizedExperiment object
+
+The `r Biocpkg("SummarizedExperiment")` object emerges as a pivotal structure, integrating drug response data with essential metadata. This versatile container plays a central role in the storage of information related to drugs, cell lines, and experimental conditions, providing a comprehensive foundation for nuanced analysis within the gDR.
+
+The `r Biocpkg("SummarizedExperiment")` object in gDR comprises four essential components:
```
Comment (gladkia) [inline vignettes/gDR-data-model.Rmd:63]: minor: consider: `comprises` => `contains`

**Review by gladkia — APPROVED** [review]

File: vignettes/gDR-data-model.Rmd (hunk 1 above)
Comment (bczech) [reply vignettes/gDR-data-model.Rmd:83]: Thanks, done

File: vignettes/gDR-data-model.Rmd (hunk 2 above)
Comment (bczech) [reply vignettes/gDR-data-model.Rmd]: thanks, done

**Pull request merged**